docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -154,6 +154,32 @@ for instance:
     - if you have extended `CountryGridFactory`, revise your changes because class changed its namespace
     - if you have extended `CountryFormType`, revise your changes – new fields are available
     - if you have extended `CountryController` revise your changes – `new` and `edit` actions were added
+- Microservices has been removed and their funcionality has been moved to framework ((#793)[https://github.com/shopsys/shopsys/pull/793/]):
+    - classes from microservices has been moved to `shopsys/framework` package under `Product\Search` namespace
+    - definitions of indexes has been moved to project folder `src/Shopsys/ShopBundle/Resources/definition`, copy them to your project as well
+        - add `shopsys.elasticsearch.structure_dir: '%shopsys.resources_dir%/definition/'` to your `paths.yml` config file
+    - Symfony commands for Elasticsearch management has changed their namespace from `shopsys:microservice:product-search` to `shopsys:product-search`
+        - update your `build.xml` and `build-dev.xml` to reflect this change
+        - if you have extended those command in your project, update their class and file names and `$defaultName` property appropriately
+    - `StructureManager` has been moved to `packages/framework/src/Component/Elasticsearch` and renamed to `ElasticsearchStructureManager`
+        - its methods `getIndexName`, `createIndex`, `deleteIndex` and `getDefinition` now accept second mandatory parameter `index`
+            - this parameter is for distinguishing different use cases like Product search
+    - in production you will need to run `product-search-recreate-structure` Phing target while next build to create indexes again with new name
+        - after that remove previous indexes used for Product search, so they do not consume any memory
+    - `ProductSearchExportFacade` has been moved from `Product/ProductSearchExport` to `Product/Search/Export`
```

Comment:
I wouldnt mention 169-171 like this, first of all I would use FQN if you are talking about it, and I think that they have benn moved isnt really a upgrade instruction, I would say that you wwant to say something like "If you were extending one of these Classes in namespace `Product/ProductSearchExport` you now need to specify different FQN while extending.

Reply:
I have updated the description